lib/player.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Player
+  attr_reader :bowl
+
+  def roll
+    'What is your roll?'
+    @bowl = $stdin.gets.chomp.to_i
+  end
+end
```

Comment:
Is a player class required? Does a player usually have to input their score themselves?

Reply:
This was an earlier strategy... should definitely delete player haha